content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-contract.md

```diff
@@ -66,15 +65,45 @@ class MarsVoucherContract : Contract {
 }
 ```
 
-### Add commands
+### Connect the `MarsVoucherContract` to the `MarsVoucher` state
+
+After creating the contract class in a CorDapp, you must connect the contract to its correlating state. Add the `@BelongsToContract` annotation *in the state class* to establish the relationship between a state and a contract. Without this, your state does not hold a relationship to the contract that is used to verify it.
 
-Commands indicate the transaction's intent — what type of actions performed by the state the contract can verify.
+1. Open your `MarsVoucher` class.
+2. Insert the `@BelongsToContract` annotation with the `MarsVoucherContract` just before the definition of the `MarsVoucher` data class.
+
+Transactions involving the `MarsVoucher` state are now verified using the `MarsVoucherContract`.
+
+```kotlin
+package net.corda.missionMars.states
+
+import com.google.gson.Gson
+import net.corda.v5.application.identity.AbstractParty
+import net.corda.v5.application.identity.Party
+import net.corda.v5.application.utilities.JsonRepresentable
+import net.corda.v5.ledger.UniqueIdentifier
+import net.corda.v5.ledger.contracts.LinearState
+
+@BelongsToContract(MarsVoucherContract::class)
+data class MarsVoucher (
+        val voucherDesc : String,//For example: "One voucher can be exchanged for one ticket to Mars"
+        val issuer: Party, //The party who issued the voucher
+        val holder: Party, //The party who currently owns the voucher
+        override val linearId: UniqueIdentifier,//LinearState required variable
+) ...
+```
 
-1. Add the `Commands : CommandData` interface declaration.
+### Define commands
 
-2. Inside the interface, add the `Issue` class that implements `Commands`.
+[Commands](../../../../../../en/platform/corda/5.0-dev-preview-1/cordapps/key-concepts/key-concepts-transactions.html#commands) are built into a transaction to indicate its intent. They control the type of actions performed to the state that the contract can verify.
 
-   The `Issue` command is used to create the Mars voucher.
+Each command is associated with a list of signers. The public keys listed in a contract's commands indicate the transaction's required signers.
+
+In the `MarsVoucherContract`, you need a command that issues the `MarsVoucher`.
```

Comment:
It issues an instance of a MarsVoucher - creating a new state on the ledger... it doens't issue <i>the</i> MarsVoucher